Prog/2022/Sales_affordability_2016_20.sas

```diff
@@ -0,0 +1,383 @@
+ 	%dollar_convert(PI_Repeat2018,PI_Repeat2018r,2018,2020,series=CUUR0000SA0L2);
+	%dollar_convert(PI_Repeat2019,PI_Repeat2019r,2019,2020,series=CUUR0000SA0L2);
+	%dollar_convert(PI_Repeat2020,PI_Repeat2020r,2020,2020,series=CUUR0000SA0L2);
+
+	*calculate monthly PITI (Principal, Interest, Taxes and Insurance) for Repeat Homebuyer (25% of PI = TI);
+	if sale_yr=2016 then PITI_Repeat=PI_Repeat2016r*1.25;
+	if sale_yr=2017 then PITI_Repeat=PI_Repeat2017r*1.25;
+	if sale_yr=2018 then PITI_Repeat=PI_Repeat2018r*1.25;
+	if sale_yr=2019 then PITI_Repeat=PI_Repeat2019r*1.25;
+	if sale_yr=2020 then PITI_Repeat=PI_Repeat2020r*1.25;
+
+
+	/*Here are numbers for Average Household Income at the city level. 2016-20 ACS 
+	Using tables B19025(B,H, I) and B11001X (B,H,I)
+	Black	NH-White	Hispanic	AIOM	 
+	72915	 194743		120441 	 	 		*/
+
+
+	if PITI_First <= (194743 / 12*.28) then white_first_afford=1; else white_first_afford=0; 
+		if PITI_Repeat <= (194743/ 12 *.28) then white_repeat_afford=1; else white_repeat_afford=0; 
+	if PITI_First <= (72915 / 12 *.28) then black_first_afford=1; else black_first_afford=0; 
+		if PITI_Repeat <= (72915 / 12 *.28) then black_repeat_afford=1; else black_repeat_afford=0; 
+	if PITI_First <= (120441 / 12*.28) then hispanic_first_afford=1; else hispanic_first_afford=0; 
+		if PITI_Repeat <= (120441 / 12*.28 ) then hispanic_repeat_afford=1; else hispanic_repeat_afford=0; 
+	/*if PITI_First <= (76271 / 12*.28 ) then aiom_first_afford=1; else aiom_first_afford=0; 
+		if PITI_Repeat <= (76271 / 12*.28 ) then aiom_repeat_afford=1; else aiom_repeat_afford=0; 
+	*/
+
+
+	total_sales=1;
+
+	label 	PITI_First = "Principal, Interest, Tax and Insurance for FT Homebuyer"
+			PITI_Repeat = "Principal, Interest, Tax and Insurance for Repeat Homebuyer"
+			white_first_afford = "Property Sale is Affordable for FT White Owners"
+			black_first_afford = "Property Sale is Affordable for FT Black Owners"
+			hispanic_first_afford = "Property Sale is Affordable for FT Hispanic Owners"
+			/*AIOM_first_afford = "Property Sale is Affordable for FT Owners of Asian, Pacific Islander, American Indian, Alaskan Native Descent, Other, Two or More Races"*/
+			white_repeat_afford = "Property Sale is Affordable for Repeat White Owners"
+			black_repeat_afford = "Property Sale is Affordable for Repeat Black Owners"
+			hispanic_repeat_afford = "Property Sale is Affordable for Repeat Hispanic Owners"
+			/*AIOM_repeat_afford = "Property Sale is Affordable for Repeat Owners of Asian, Pacific Islander, American Indian, Alaskan Native Descent, Other, Two or More Races"*/
+;
+
+
+run;
+proc print data= create_flags (obs=25);
+var ward2012 saleprice PITI_FIRST PITI_repeat white_first_afford black_first_afford hispanic_first_afford /*AIOM_first_afford*/;
+run; 
+
+proc freq data=create_flags; 
+tables white_first_afford black_first_afford hispanic_first_afford /*AIOM_first_afford*/; 
+run;
+*testing*;
```

Comment:
Not clear to me what this is testing or what I should be looking for in the outputs.